### PR TITLE
add io.github.ddanilov.soundscape

### PR DESCRIPTION
**Pull request by ddanilov**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: io.github.ddanilov.soundscape.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+app-id: io.github.ddanilov.soundscape
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: soundscape
+finish-args:
+  # directory access
+  - --filesystem=host
```
Comment (hfiguiere) [inline io.github.ddanilov.soundscape.yaml:8]: this is a very broad permission

File: io.github.ddanilov.soundscape.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - --share=ipc
+  - --socket=x11
+  # audio
+  - --socket=pulseaudio
+rename-appdata-file: soundscape.metainfo.xml
+rename-desktop-file: soundscape.desktop
+rename-icon: soundscape
+modules:
+  - name: soundscape
+    buildsystem: cmake
```
Comment (hfiguiere) [inline io.github.ddanilov.soundscape.yaml:20]: cmake-ninja?

**Comment by hfiguiere** [comment]
bot, build io.github.ddanilov.soundscape

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/io.github.ddanilov.soundscape.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
Validation of files failed
```

File: io.github.ddanilov.soundscape.yaml (hunk 2 above)
Comment (ddanilov) [reply io.github.ddanilov.soundscape.yaml]: can also be `-ninja`, I do not see any difference there

**Comment by ddanilov** [comment]
> ```
> builddir/files/share/appdata/io.github.ddanilov.soundscape.appdata.xml: FAILED:
> • tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
> • tag-missing           : <release> required
> Validation of files failed
> ```

should be fixed now with the updated PR

File: io.github.ddanilov.soundscape.yaml (hunk 2 above)
Comment (hfiguiere) [reply io.github.ddanilov.soundscape.yaml]: then use it, it's faster. `cmake` only exist as a fallback

**Comment by bbhtt** [comment]
bot, build io.github.ddanilov.soundscape

**Comment by ddanilov** [comment]
> Build [68791 successful](https://buildbot.flathub.org/#/builders/6/builds/68791) To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/51438/io.github.ddanilov.soundscape.flatpakref
> ```

looks good :+1:

**Pull request closed without merging**